apps/arclight/src/app/videoPlayerUrl/VideoPlayer.css

```
@@ -0,0 +1,444 @@
+  -webkit-transition: opacity 0.1s;
+  -moz-transition: opacity 0.1s;
+  -o-transition: opacity 0.1s;
+  transition: opacity 0.1s;
+}
+.video-js .vjs-dock-text {
+  left: 0;
+  margin: 0;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.8);
+  background: -webkit-linear-gradient(
+    to bottom,
```

Biome (1.9.4)

[error] Unexpected nonstandard direction

You should fix the direction value to follow the syntax.
See MDN web docs for more details.

(lint/correctness/noInvalidDirectionInLinearGradient)